### PR TITLE
feat(license): use agora order for  vps

### PR DESCRIPTION
**Pull request by lizardK**
| Question         | Answer
| ---------------- | ---
| Branch?          |  `develop`
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-5669
| License          | BSD 3-Clause

## Description

Use new order flow when ordering licenses for VPS new range

ref: MANAGER-5669

## Related

- [ ] ~TRANS-29068~

### CODE REVIEW

**Review by frenautvh — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/license/order/license-order.html [hunk 1]
```diff
@@ -2,16 +2,16 @@
     <h1 data-translate="license_order_title"></h1>
 </header>
 
-<div class="text-center" data-ng-if="loaders.ips">
+<div class="text-center" data-ng-show="loaders.ips">
```
Comment (marie-j) [inline packages/manager/apps/dedicated/client/app/license/order/license-order.html:5]: `ng-if` causing trouble ?

File: packages/manager/apps/dedicated/client/app/dedicatedUniverseComponents/duration-iso8601/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+import translate from 'angular-translate';
+import ngTranslateAsyncLoader from '@ovh-ux/ng-translate-async-loader';
+import durationISO8601Filter from './duration-iso8601.filter';
+
+const moduleName = 'durationISO8601';
+
+angular
+  .module(moduleName, [ngTranslateAsyncLoader, translate])
+  .filter('durationISO8601', durationISO8601Filter)
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/dedicatedUniverseComponents/duration-iso8601/index.js:10]: This filter could be set in this package: https://github.com/ovh/manager/tree/master/packages/manager/modules/filters

File: packages/manager/apps/dedicated/client/app/license/order/agoraOrder/license-agoraOrder.service.js [hunk 3]
```diff
@@ -25,18 +26,117 @@ class LicenseAgoraOrder {
...
+    /* this.licenseTypeToCatalog = {
+      CLOUDLINUX: 'licenseCloudLinux',
+      CPANEL: 'cpanel',
+      DIRECTADMIN: 'licenseDirectadmin',
+      PLESK: 'plesk',
+      SQLSERVER: 'licenseSqlServer',
+      VIRTUOZZO: 'licenseVirtuozzo',
+      WINDOWS: 'licenseWindows',
+      WORKLIGHT: 'licenseWorklight',
+    }; */
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/license/order/agoraOrder/license-agoraOrder.service.js:39]:
```suggestion
```

File: packages/manager/apps/dedicated/client/app/license/order/license-order-classic.controller.js [hunk 4]
```diff
@@ -0,0 +1,516 @@
...
+  MAX_CPANEL_PRO_ACCOUNTS,
+  MAX_CPANEL_ADMIN_ACCOUNTS,
+  LICENCE_TYPES,
+} from './license-order.constants';
+
+angular
+  .module('Module.license')
+  .controller(
+    'LicenseOrderClassicCtrl',
+    (
```
Comment (JeremyDec) [inline packages/manager/apps/dedicated/client/app/license/order/license-order-classic.controller.js:29]:
```suggestion
    /* @ngInject */ (
```

**Comment by ovh-cds** [comment]
CDS Report build#16194.0 ✘  
* 
  * Build ✘

File: packages/manager/apps/dedicated/client/app/license/order/license-order.html (hunk 1 above)
Comment (lizardK) [reply packages/manager/apps/dedicated/client/app/license/order/license-order.html:5]: As it changes a lot, avoid DOM Reflow

**Comment by ovh-cds** [comment]
CDS Report translator-push#11708.0 ✘  
* 
  * push ✘

**Comment by ovh-cds** [comment]
CDS Report translator-push#11709.0 ✘  
* 
  * push ✘

**Comment by marie-j** [comment]
Rebase to change target branch

**Review by antleblanc — APPROVED** [review]

**Pull request merged**